lib/federated/generator.rb

```diff
@@ -12,11 +12,20 @@ def create!(options={})
       relayable = build(options)
       if relayable.save!
         logger.info "user:#{@user.id} dispatching #{relayable.class}:#{relayable.guid}"
+        add_root_author(relayable)
         Postzord::Dispatcher.defer_build_and_post(@user, relayable, @dispatcher_opts)
         relayable
       end
     end
 
+    def add_root_author(relayable)
+      return unless relayable.parent.respond_to?(:root) && relayable.parent.root
+      # Comment post is a reshare, include original author in subscribers
+      root_post = relayable.parent.root
+      @dispatcher_opts[:additional_subscribers] ||= []
+      @dispatcher_opts[:additional_subscribers] << Person.find(root_post.author)
```

Comment:
`author` should return a `Person` instance already, the `Person.find` is redundant.